src/pandroid/app/src/main/java/com/panda3ds/pandroid/data/config/GlobalConfig.java

```diff
@@ -0,0 +1,61 @@
+package com.panda3ds.pandroid.data.config;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.panda3ds.pandroid.app.PandroidApplication;
+import com.panda3ds.pandroid.utils.Constants;
+
+import java.io.Serializable;
+
+public class GlobalConfig {
+    private static SharedPreferences data;
+
+    public static void initialize() {
+        data = PandroidApplication.getAppContext()
+                .getSharedPreferences(Constants.PREF_GLOBAL_CONFIG, Context.MODE_PRIVATE);
+    }
+
+    private static <T extends Serializable> T get(Key<T> key) {
+        Serializable value;
+        if (key.defValue instanceof String) {
+            value = data.getString(key.name, (String) key.defValue);
+        } else if (key.defValue instanceof Integer) {
+            value = data.getInt(key.name, (int) key.defValue);
+        } else if (key.defValue instanceof Boolean) {
+            value = data.getBoolean(key.name, (Boolean) key.defValue);
+        } else if (key.defValue instanceof Long) {
+            value = data.getLong(key.name, (Long) key.defValue);
```

Comment:
Is there a reason we use the object types (`Long`, `Boolean`) on some cases and the primitive on others (`int`, `float`). If possible let's try to be consistent and use the primitive types for the numbers (`int`, `long`, `boolean`, `float`). `String` is fine. 

For example, looking at the signature of getLong: `long getLong(String key, long defValue)` it returns `long` and not `Long`

Reply:
i know, but java dont support instanceof for primitives: 
![image](https://github.com/wheremyfoodat/Panda3DS/assets/97042217/dc992008-1488-4ee6-9212-dfe4c6f54c8d)